requirements-dev-thirdparty.txt

```diff
@@ -7,8 +7,9 @@ git+https://github.com/asdf-format/asdf-wcs-schemas
 
 # Use nightly Astropy dev builds
 git+https://github.com/astropy/asdf-astropy
---extra-index-url https://pypi.anaconda.org/astropy/simple
+--extra-index-url https://pypi.anaconda.org/liberfa/simple
 pyerfa>=0.0.dev0
```

Comment:
Thanks. Perhaps you can help me understand why `pyerfa` is here in the first place.

Since we don't use this directly (it's not listed as a dependency and not imported), why do we need it in the devdeps?  I don't see this listed in the "Install Pre-built Development Version" docs for astropy:
https://docs.astropy.org/en/latest/install.html#install-pre-built-development-version
Is there some place that documents this as "recommended" and explains why the above docs are incomplete?

Reply:
It does not happen often but there are cases where sometimes pyerfa broke astropy, and one would need pyerfa-dev for a bit until pyerfa can release. astropy used to bundle it but not anymore.

That said, if you really don't like pyerfa-dev, we can also just remove `pyerfa>=0.0.dev0` and be done with it, but I did not add it there to begin with, I am just fixing it because without this patch, `pyerfa>=0.0.dev0` in that file is a lie.

Also, looks like romancal also installs pyerfa-dev for devdeps, so it all depends on what you want and if you want it uniformly across the board.

Reply:
Thanks. That's helpful to understand why it was added in https://github.com/spacetelescope/jwst/pull/8701

I'm inclined to remove it across the board. @zacharyburnett what do you think?

Reply:
Oh wait, so I did add it.... 😆  Ooops

Yes, I am okay with it if you want to remove it.

Reply:
Done.